joint_calling/upload_processor.py

```diff
@@ -0,0 +1,88 @@
+""" This function prepares gVCF's uploaded to GCS, based off sample status
+    logged in a database, for further QC and downstream analysis.
+    The upload processor will determine when samples should be added
+    to existing MatrixTables where appropriate and which MatrixTables
+    they should be combined with in this case. Following a successful
+    run all uploaded files will be moved to archival storage."""
+
+from typing import Tuple, List
+from google.cloud import storage
+from google.api_core.exceptions import NotFound
+
+
+def copy_files(
+    sample_files: List[str], source_bucket_name: str, destination_bucket_name: str
+) -> Tuple[List[str], List[str]]:
+
+    """Given a list of file names, a source bucket and
```

Comment:
Does the list of file names contain the bucket name? This is not obvious, so it might make sense to spell this out.

Also consider splitting out the documentation for each parameter, like https://hail.is/docs/0.2/_modules/hail/context.html#init.

Reply:
Great point! Have updated comments to clarify.